otterai/llm_client.py

```diff
@@ -0,0 +1,21 @@
+import os
+import httpx
+from langchain_openai import ChatOpenAI
+
+class LLMClient:
+    _instance = None
+
+    def __new__(cls) -> 'LLMClient':
```

Comment:
🔄 The '__new__' method is overridden; ensure it's correctly handling inheritance and instance creation.